std/assembly/dataview.ts

```diff
@@ -0,0 +1,127 @@
+import { HEADER_SIZE } from "./internal/arraybuffer";
+
+@inline
+function get<T>(buffer: ArrayBuffer, byteOffset: i32, littleEndian: boolean): T {
+  const result = load<T>(changetype<usize>(buffer) + byteOffset, HEADER_SIZE);
```

Comment:
in AS better use `var` instead `const` if this variable can't compute in compile time or store in data section

Reply:
Good to know, will fix 👍 

Do you mind explaining why? ☺️ it is also better than `let`?

Reply:
It better ask to @dcodeIO

Reply:
As a rule of thumb, `const` currently behaves pretty much like `static const` in other languages, indicating storage in static memory. I'm considering changing that, though.